src/ListGroup/ListGroup.stories.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { defaultTemplate } from '../../storybook/decorators/storyTemplates'
+import { ListGroup, ListGroupItem } from '../index.js'
+
+const stories = storiesOf('ListGroup', module)
+
+const description = (
+  <p>
+    This component is based on React Bootstrap ListGroup component. See{' '}
+    <a href="https://react-bootstrap.github.io/components.html#listgroup">
+      React Bootstrap Docs
+    </a>{' '}
+    for complete ListGroup component documentation.
+  </p>
+)
+
+stories.addDecorator(
+  defaultTemplate({
+    title: 'ListGroup',
+    documentationLink:
+      'http://www.patternfly.org/pattern-library/widgets/#list-group',
+    description: description
+  })
+)
+
+stories.addWithInfo('Basic Example', '', () => (
+  <ListGroup>
+    <ListGroupItem>Cras justo odio</ListGroupItem>
+    <ListGroupItem>Dapibus ac facilisis in</ListGroupItem>
+    <ListGroupItem>Morbi leo risus</ListGroupItem>
+    <ListGroupItem>Porta ac consectetur ac</ListGroupItem>
+    <ListGroupItem>Vestibulum at eros</ListGroupItem>
+  </ListGroup>
+))
+
+stories.addWithInfo('Badges', '', () => (
+  <ListGroup>
+    <ListGroupItem>
+      Cras justo odio <span className="badge">14</span>
```

Comment:
I think Badge should be a component:
https://react-bootstrap.github.io/components.html#badges

Reply:
+1... i'll update this PR shortly